tests/testthat/test-load_db.R

```diff
@@ -0,0 +1,17 @@
+# setup
+dbname <- tempfile(fileext = ".db")
+db_connection_origin <- DBI::dbConnect(RSQLite::SQLite(), dbname)
+
+test_that("database gets loaded", {
+  
+  expect_equal(file.size(dbname), 0)
+  
+  load_db(
+    db_connection = db_connection_origin,
+    csv_path_dir = "dbtest",
+    cdm_metadata = concePTION_metadata_v2,
```

Comment:
concePTION_metadata_v2 is undefined. should it be loaded first under setup?
data("concePTION_metadata_v2")

Reply:
This file is added to the package, under `data/`; it is part of this PR. When the package is in the environment, the data can be accessed with the variable `concePTION_metadata_v2`, as is done here.